src/components/Transactions/TransactionDetail.tsx

```diff
@@ -134,9 +134,7 @@ export default function TransactionDetail({
                   <Typography color={theme.palette.text.secondary}>
                     {t('app.transactionDetail.dateAndTime')}
                   </Typography>
-                  <Typography fontWeight={600}>
-                    {transactionData.dateTime.toDateString()}
-                  </Typography>
+                  <Typography fontWeight={600}>{transactionData.dateTime}</Typography>
```

Comment:
The same as above

Reply:
Small problem.. use this, you need to have a date object, to make a date object, you need to have a valid string (so the string that is returned from the be mock, which is "string" doesn't work, so it throws an error). What should I do? Implement it with a fake date (maybe in the converter util), or leave it as it is?

Reply:
If you have a string you can cast in this way
```
const serviceValue = "21/03/2024".split("/")
const dateformat = new Date(serviceValue[2], serviceValue[1], serviceValue[0]);
```
then
```
const dateHumnaFriendly = new Intl.DateTimeFormat(LANG).{
    dateStyle: 'short',
    timeStyle: 'short'
  }.format(dateformat)
```

Reply:
The problem is that the BE returns a string with "string" as a value, which is not accepted by the new Date constructor, even if I split it, it'd still give me an empty value, which isn't liked by the constructor.